src/rantanplan/core.py

```diff
@@ -725,8 +725,20 @@ def generate_liaison_positions(syllables, liaison):
     liaison_indices = [
         index for index, position in enumerate(positions) if position
     ]
+    # Priorize single liaisons
+    non_single_liaisons = []
     for combination in combinations:
         liaison_positions = [0] * len(positions)
         for index, liaison_index in enumerate(liaison_indices):
             liaison_positions[liaison_index] = combination[index]
-        yield liaison_positions
+        if has_single_liaisons(liaison_positions):
+            yield liaison_positions
+        else:
+            non_single_liaisons.append(liaison_positions)
+    for liaison_position in non_single_liaisons:
+        yield liaison_position
+
+
+def has_single_liaisons(liaisons):
+    print('\n----------------------------', '\n', liaisons, '\n----------------------------')
+    return not any(i == j == 1 for i, j in zip(liaisons, liaisons[1:]))
```

Comment:
The `print` must go out. And it's missing some basic documentation. This function checks whether the list `liaisons` has two or more consecutive `1`'